src/solvers/qbf/qbf_bdd_core.cpp

```diff
@@ -227,10 +240,10 @@ propt::resultt qbf_bdd_coret::prop_solve()
         matrix->nodeCount() << " nodes" << std::endl;
       #endif
 
-      *matrix = matrix->UnivAbstract(*bdd_variable_map[var]);
+      *matrix=matrix->UnivAbstract(*bdd_variable_map[var]);
     }
     else
-      throw ("Unquantified variable");
+      throw("Unquantified variable");
```

Comment:
Nit: parentheses are unnecessary (and we don't usually use them with `throw` - applies below as well.